src/components/ClassCard/ClassCard.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import "./ClassCard.css";
+import { IoRemoveCircle } from "react-icons/io5";
+
+export default function ClassCard({
+  id,
+  name,
+  duration,
+  intervalTime,
+  onRemoveClassClick,
+}) {
+  return (
+    <section className="ClassCard">
+      <h2>{name}</h2>
+      <div className="ClassCard__input">
```

Comment:
Why do you put `__input` in the class name? I don't see anything that could have to do with an input?

Reply:
True, changed it to info.